### ISSUE TITLE
[Doc] Document the removal of parsec device using the CLI

### ISSUE THREAD

**Issue by FirelightFlagboy** [Scille/parsec-cloud#8448]
The CLI provide the command `device remove` to remove a local device, but that's not documented.

**Comment by mmmarcos** [comment]
For the end-user we'll add a specific function in the GUI based on bindings: https://github.com/Scille/parsec-cloud/issues/8337

Regarding CLI documentation, this is required, but it would be nice to do it automatically (based on source code). So we should open another issue, or maybe update this issue's title.

**Comment by AureliaDolo** [comment]
`device remove` has been renamed to `device forget-local` is this issue still relevant ?

**Comment by FirelightFlagboy** [comment]
Yes, the goal of the issue hasn't changed, we would like to document how to remove a local device
